Hazel/src/Hazel/Renderer/OrthographicCameraController.cpp

```diff
@@ -54,7 +54,18 @@ namespace Hazel {
 
 		m_Camera.SetPosition(m_CameraPosition);
 
-		m_CameraTranslationSpeed = m_ZoomLevel;
+		//m_CameraTranslationSpeed = m_ZoomLevel;
+	}
+
+	void OrthographicCameraController::SetZoomLevel(float level)
+	{
+		HZ_CORE_ASSERT(level > 0.0f, "Zoom Level have to be bigger than 0.0f!");
```

Comment:
Zoom Level has to be bigger than 0.0f!

Reply:
I've updated that in the local repo.

Reply:
Update the PR as well?

Reply:
I wanted to wait for other change suggestions and update them all at once.
Anyway, PR is updated.